bin/lesson_check.py

```diff
@@ -82,7 +86,9 @@
     'language-python',
     'language-r',
     'language-shell',
-    'language-sql'
+    'language-sql',
+    'bash',
+    'python'
```

Comment:
I don't think these should be here since they are not actually available.  Looks like the lesson_check is out of date in general, https://github.com/carpentries/lesson-example/blob/gh-pages/bin/lesson_check.py#L77 (I'm not suggesting you should update it though!)

Reply:
Good call -- fixed, 3a1500e